provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1479,8 +1477,13 @@ func vpcCniProperties(kubeconfig bool) map[string]schema.PropertySpec {
 		},
 		"enablePodEni": {
 			TypeSpec: schema.TypeSpec{Type: "boolean"},
-			Description: "Specifies whether to allow IPAMD to add the `vpc.amazonaws.com/has-trunk-attached` label to" +
-				"the node if the instance has capacity to attach an additional ENI. Default is `false`.",
+			Description: "Specifies whether to allow IPAMD to add the `vpc.amazonaws.com/has-trunk-attached` label to " +
+				"the node if the instance has capacity to attach an additional ENI. Default is `false`. " +
+				"If using liveness and readiness probes, you will also need to disable TCP early demux.",
+		},
+		"disableTcpEarlyDemux": {
+			TypeSpec:    schema.TypeSpec{Type: "boolean"},
+			Description: "Allows the kubelet's liveness and readiness probes to connect via TCP when pod ENI is enabled. This will slightly increase local TCP connection latency.",
```

Comment:
It's a little unfortunate, but for consistency, lets line wrap this similar to other descriptions in this file.  (A little surprised we aren't getting a int failure on this).

Reply:
done